packages/core/content-releases/admin/src/components/tests/CMReleasesContainer.test.tsx

```diff
@@ -28,42 +23,48 @@ jest.mock('react-router-dom', () => ({
 }));
 
 describe('CMReleasesContainer', () => {
-  it('should not render the container when creating an entry', () => {
+  beforeEach(() => {
     // @ts-expect-error - Ignore error
-    useCMEditViewDataManager.mockReturnValueOnce({
+    useCMEditViewDataManager.mockReturnValue({
+      isCreatingEntry: false,
+      hasDraftAndPublish: true,
+      initialData: { id: 1 },
+      slug: 'api::article.article',
+    });
+  });
+
+  it('should not render the container when creating an entry', async () => {
+    // @ts-expect-error - Ignore error
+    useCMEditViewDataManager.mockReturnValue({
       isCreatingEntry: true,
-      allLayoutData: {
-        contentType: {
-          options: {
-            draftAndPublish: true,
-          },
-        },
-      },
+      hasDraftAndPublish: true,
+      initialData: { id: 1 },
+      slug: 'api::article.article',
     });
 
     render(<CMReleasesContainer />);
 
-    const informationBox = screen.queryByRole('complementary', { name: 'Releases' });
-    expect(informationBox).not.toBeInTheDocument();
+    await waitFor(() => {
+      const informationBox = screen.queryByRole('complementary', { name: 'Releases' });
+      expect(informationBox).not.toBeInTheDocument();
+    });
```

Comment:
Could you avoid the waitFor with findByRole?

Reply:
You need to use query methods for asserting that elements are not present, because find methods rejects when there is nothing, instead of query methods that just return null.